home/api/Config.ts

```diff
@@ -1,6 +1,11 @@
+const host = 'exp.host';
+//const host = 'staging.exp.host';
+//const host = 'ed56a018.ngrok.io';
+const url = `https://${host}`;
```

Comment:
Should this be `origin`? https://www.w3.org/TR/2011/WD-html5-20110525/origin-0.h

Reply:
Whichever is the most consistent naming convention works for me. The link doesn't seem to work, can you provide a working link?

Reply:
`origin` is indeed more concise. I've updated the PR accordingly. Thanks for the review 👍